stdlib/REPL/src/TerminalMenus/util.jl

```diff
@@ -12,54 +12,43 @@
     PAGE_UP,
     PAGE_DOWN)
 
-# Enable raw mode. Allows us to process keyboard inputs directly.
-function enableRawMode(term)
+# Set `term` mode; raw=true intercepts keyboard inputs directly
+function raw!(term, raw::Bool)
```

Comment:
Isn't it possible to use directly `REPL.Terminals.raw!` here instead of wrapping it? It even seems they return the same `Bool`.